src/crate/crash/command.py

```diff
@@ -443,7 +443,9 @@ def _try_exec_cmd(self, line):
         words = line.split(' ', 1)
         if not words or not words[0]:
             return False
-        cmd = self.commands.get(words[0].lower())
+        cmd = self.commands.get(words[0].lower().strip(';'))
```

Comment:
use rstrip instead of strip. If the string starts with a semicolon it should be an error too